lib/likert/utilities/has-label.test.ts

```diff
@@ -0,0 +1,185 @@
+import { LikertOptionInterface } from '../../likert';
+import { NAMESPACE } from '../../utilities/ts/constants';
+import hasLabel from './has-label';
+
+describe(`hasLabel()`, () => {
+  it(`Handles an array with a length of three`, () => {
+    const arr: LikertOptionInterface[] = [
```

Comment:
it's good to use data providers and/or (test) helper methods in this kind of repetitive scenarios,
e.g:
https://ryanwelcher.com/2016/12/better-unit-testing-dataproviders/ 
https://blog.jphpsf.com/2012/08/30/drying-up-your-javascript-jasmine-tests/

Reply:
I've implemented this in principle, now.

I expect I'll do much more of this as I begin to refactor unit tests.